### ISSUE TITLE
General error: MODULE_NOT_FOUND

### ISSUE THREAD

**Issue by wardbeyens** [martiliones/icon-set-creator#24]
```
npm run create-appicon
    at Module._resolveFilename (node:internal/modules/cjs/loader:1039:15)
    at Module._load (node:internal/modules/cjs/loader:885:27)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (C:\Projects\PiekverbruikMonitor\node_modules\icon-set-creator\index.js:38:34)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
``` 

Error at:
```
commander_1.program
    .version(`icon-set-creator ${require('../package').version}`)
```

the require with ../package I think it should be ./package but I could be wrong

**Comment by wardbeyens** [comment]
```
> piekverbruik-monitor@0.0.1 create-appicon
> iconset create ./icon.png

node:internal/modules/cjs/loader:1042
  throw err;
  ^
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (C:\Projects\PiekverbruikMonitor\node_modules\icon-set-creator\index.js:38:34)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Projects\\PiekverbruikMonitor\\node_modules\\icon-set-creator\\index.js'
  ]
}

Node.js v18.13.0
```

**Comment by martiliones** [comment]
It looks like I had changes that I didn't push to GitHub because previous versions are importing `./package` instead of `../package`. Should be fixed in v1.1.5, let me know if it works for you
